app/com/linkedin/drelephant/executors/QuartzExecutorService.java

```diff
@@ -126,6 +125,7 @@ public void execute(AnalyticJob analyticJob) {
             JobDetail job = JobBuilder.newJob(QuartzExecutorService.ExecutorJob.class)
                     .withIdentity(constructJobKey(analyticJob.getAppId(), ExecutorJob.class.getName()))
                     .usingJobData(constructJobDataMap("analyticJob", analyticJob))
+                    .requestRecovery(true)
```

Comment:
This ensures that the executing jobs get fired again if instance goes down during their execution.